arangod/RocksDBEngine/RocksDBIterators.cpp

```diff
@@ -333,3 +333,110 @@ void RocksDBSortedAllIterator::reset() {
   TRI_ASSERT(_trx->state()->isRunning());
   _iterator->Seek(_bounds.start());
 }
+
+
+RocksDBGenericIterator::RocksDBGenericIterator(rocksdb::ReadOptions& options
+                                              ,RocksDBKeyBounds const& bounds
+                                              ,bool reverse)
```

Comment:
The S2 branch (just merged) introduced `IndexIteratorOptions` to replace the `reverse` flag and some other assorted constructor inputs. Could you switch to use that?